StyleGuide.cs

```diff
@@ -45,13 +45,11 @@ sealed class StyleGuide : IDisposable
 		/// </summary>
 		public string AnExamplePublicField;
 
-		/// <summary>
-		/// Public Property
-		///		Upper case first letter
-		///		Camel Casing
-		///		Place at top of class, after private members
-		/// </summary>
-		public string PublicProperty { get; set; }
+		// /// <summary>
+		// /// Public Property
+		// /// Don't use. If you need to mutate the class, consider a method. If you need a Public Get, use the Lambda style.

```

Comment:
Formatting (indentation) inconsistent.
Whilst this is a style guide and so this development practice comment could be unrelated, i think its worthwhile having. Please reword to not replace the original style guide comments, and make this an advisory

Reply:
Yeah I've changed to an advisory and tried to bring the tone closer to the rest of the guide.